sql/log_event.cc

```diff
@@ -4909,19 +4909,29 @@ int Query_log_event::do_apply_event(Relay_log_info const *rli,
         }
 
         if (sqlcom_can_generate_row_events(thd->lex->sql_command) &&
-            thd->get_row_count_func() > 0 &&
```

Comment:
@dutow, just to recheck, is this change expected to be here? This looks like one of the conflicts in 8.0.31 merge.

Reply:
No, this and the related change in sql change shouldn't be there, good catch, I'll revert this.

Reply:
Thanks!